packages/reporters/src/NpmLogReporter.ts

```diff
@@ -107,25 +137,29 @@ export class NpmLogReporter implements Reporter {
   private logTargetEntry(entry: LogEntry<TargetStatusEntry | TargetMessageEntry>) {
     const colorFn = colors[entry.level];
     const data = entry.data!;
+    const pkg = data['package'];
+    const task = data['task'];
 
     if (isTargetStatusLogEntry(data)) {
       const { hash, duration } = data;
+      const pkgColor = getColorForPkg(pkg);
+      const pkgTask = this.options.grouped ? `${pkgColor(pkg)} ${chalk.cyan(task)}` : "";
 
       switch (data.status) {
         case "running":
-          return this.printEntry(entry, colorFn(`${colors.ok("➔")} start`));
```

Comment:
I think the change should be inside the printEntry function? It already contains the "pkgTask" stuff that's repeated here.

Reply:
Ok, the message will have a slightly different format for some statuses, but I think that's ok